### PR TITLE
Add a Sphinx extension to generate documentation coverage information

### PR DESCRIPTION
**Pull request by Gallaecio**

### CODE REVIEW

File: docs/_ext/scrapycoverage.py [hunk 1]
```diff
@@ -0,0 +1,291 @@
+"""
+    scrapycoverage
+    ~~~~~~~~~~~~~~
+
+    Check Python modules and C API for coverage.
+
+    Fork of sphinx.ext.coverage with the following changes:
```
Comment (kmike) [inline docs/_ext/scrapycoverage.py]: Hey! How hard would it be to either get changes merged to sphinx, or create a separate package for this (we can use scrapy github org)? It looks risky to have such code in `docs/_ext`, with not tests, etc.

Comment (Gallaecio) [reply docs/_ext/scrapycoverage.py]: I’ll try to get it merged upstream, with a few other changes.

Comment (kmike) [reply docs/_ext/scrapycoverage.py]: That'd be perfect!

**Comment by Gallaecio** [comment]
Changes got merged upstream! :tada: 

Now I’ll wait for Sphinx 2.1 to be released, and redo this patch on top of it.

**Comment by Gallaecio** [comment]
Done.

**Comment by kmike** [comment]
Thanks @Gallaecio for doing it the "hard way" (making it a part of Sphinx)!

**Pull request merged**